index.html

```diff
@@ -5879,6 +5879,19 @@ <h2 id="AudioWorklet-concepts">
               "#widl-AudioWorkletGlobalScope-registerProcessor-void-DOMString-name-VoidFunction-processorCtor">
               registerProcessor</a> method is called.
             </dd>
+            <dt>
+              <dfn>node name to parameter descriptor map</dfn>
+            </dt>
+            <dd>
+              Belongs to <a>BaseAudioContext</a>. This contains a collection of
+              <a>AudioParamDescriptor</a>s with an associated string key of
+              node name. This internal storage is populated from <a>node name
```

Comment:
I would reference the preceding definition of the string key so that it is clearer that the keys are the same in both internal maps.